db/institutions.yml

```diff
@@ -420,4 +420,12 @@ uduisburgessen:
   state: North Rhine-Westphalia
   street: Universitaetsstrasse 7
   zip: 45117
-
+drexelu:
```

Comment:
Drexel is already an entry but it needs to have departments and schools added.  Please could you rebase and then do this over?  Thanks